Bug: zone.js import not found in versions greater than 0.13.1

### Please provide the environment you discovered this bug in.

M1 Mac, Node 20.5.1, `zone.js` 0.13.2 and 0.13.3

### Which area/package is the issue in?

astro-angular

### Description

[Recent changes](https://github.com/angular/angular/commit/4798ec41668d47fd5e1504c61d96d5e56dcff345) to zone.js exports and bundling are breaking the Astro integration. This happened when adding the plugin with `pnpm astro add @analogjs/astro-angular` because the `zone.js` version isn't pinned.


### Please provide the exception or error you saw

```
✘ [ERROR] Could not resolve "zone.js/dist/zone.js"
    node_modules/@analogjs/astro-angular/src/client.js:1:7:
      1 │ import 'zone.js/dist/zone.js';
        ╵        ~~~~~~~~~~~~~~~~~~~~~~
  The module "./bundles/zone.js.umd.js" was not found on the file system:
    node_modules/zone.js/package.json:91:17:
      91 │       "default": "./bundles/*.umd.js"
         ╵                  ~~~~~~~~~~~~~~~~~~~~
```


### Other information

Removing `.js` from the import fixes the issue and does not break backwards compatibility. Otherwise we could pin the version.

### I would be willing to submit a PR to fix this issue

- [X] Yes
- [ ] No

--- a/package.json
+++ b/package.json
@@ -79,7 +79,7 @@
     "ufo": "^1.1.2",
     "xhr2": "^0.2.1",
     "zod": "^3.21.4",
-    "zone.js": "0.13.1"
+    "zone.js": "^0.13.3"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "16.2.0",

--- a/packages/astro-angular/src/client.ts
+++ b/packages/astro-angular/src/client.ts
@@ -1,4 +1,4 @@
-import 'zone.js/dist/zone.js';
+import 'zone.js';
 import {
   EnvironmentProviders,
   Provider,

--- a/packages/astro-angular/src/server.ts
+++ b/packages/astro-angular/src/server.ts
@@ -1,4 +1,4 @@
-import 'zone.js/bundles/zone-node.umd.js';
+import 'zone.js/node';
 import type {
   ComponentMirror,
   EnvironmentProviders,

--- a/packages/platform/src/lib/ssr/ssr-build-plugin.ts
+++ b/packages/platform/src/lib/ssr/ssr-build-plugin.ts
@@ -3,15 +3,6 @@ import { normalizePath, Plugin } from 'vite';
 export function ssrBuildPlugin(): Plugin {
   return {
     name: 'analogjs-ssr-build-plugin',
-    config(_config) {
-      return {
-        resolve: {
-          alias: {
-            'zone.js/node': 'zone.js/bundles/zone-node.umd.js',
-          },
-        },
-      };
-    },
     transform(code, id) {
       if (id.includes('platform-server')) {
         code = code.replace(/global\./g, 'globalThis.');

--- a/packages/vite-plugin-angular/setup-vitest.ts
+++ b/packages/vite-plugin-angular/setup-vitest.ts
@@ -1,6 +1,6 @@
 import 'zone.js';
-import 'zone.js/dist/sync-test';
-import 'zone.js/dist/proxy';
+import 'zone.js/plugins/sync-test';
+import 'zone.js/plugins/proxy';
 import 'zone.js/testing';
 
 /**
@@ -23,10 +23,10 @@ const SyncTestZoneSpec = Zone['SyncTestZoneSpec'];
 const ProxyZoneSpec = Zone['ProxyZoneSpec'];
 
 if (SyncTestZoneSpec === undefined) {
-  throw new Error('Missing: SyncTestZoneSpec (zone.js/dist/sync-test)');
+  throw new Error('Missing: SyncTestZoneSpec (zone.js/plugins/sync-test)');
 }
 if (ProxyZoneSpec === undefined) {
-  throw new Error('Missing: ProxyZoneSpec (zone.js/dist/proxy.js)');
+  throw new Error('Missing: ProxyZoneSpec (zone.js/plugins/proxy.js)');
 }
 
 const env = globalThis as any;

--- a/packages/vite-plugin-nitro/src/lib/vite-plugin-nitro.ts
+++ b/packages/vite-plugin-nitro/src/lib/vite-plugin-nitro.ts
@@ -141,7 +141,7 @@ export function nitro(options?: Options, nitroOptions?: NitroConfig): Plugin[] {
                 inline: ['zone.js/node'],
                 external: ['rxjs', 'node-fetch-native/dist/polyfill', 'destr'],
               },
-              moduleSideEffects: ['zone.js/bundles/zone-node.umd.js'],
+              moduleSideEffects: ['zone.js/node'],
               renderer: normalizePath(`${__dirname}/runtime/renderer`),
               handlers: [
                 {

